pkg/esp32_sdk_lib_wifi/Makefile

```diff
@@ -1,7 +1,7 @@
 PKG_NAME=esp32_sdk_lib_wifi
 PKG_URL=https://github.com/espressif/esp32-wifi-lib
-# This is a version in the v4.4 release branch
-PKG_VERSION=cd7d14917f2c3d0ea4382f4a188cb290304faf47
+# This is the last version with the legacy system event framework
+PKG_VERSION=81768e63548385be79e7b35828832a53faba4393
```

Comment:
This commit and commit [c73ee33](https://github.com/espressif/esp32-wifi-lib/commit/c73ee337c334597378e79219c9f59525116437d2) - `Remove legacy system event framework(1948a1c7) ` are commits from the master branch that do not match the ESP-IDF 4.4 branch but are for the upcoming ESP-IDF version 5.0. The latest matching commit is [25e90ea](https://github.com/espressif/esp32-wifi-lib/commit/25e90eac783e0aeb6772d91585c671e00388126d) ([release/v4.4](25e90eac783e0aeb6772d91585c671e00388126d)).

BTW, with the ESP32-S3 port, this library is upgraded to commit [1b0fcf9](https://github.com/espressif/esp32-wifi-lib/commit/1b0fcf9eeb96ddc4d72ab5850a0bf9e95e289da4).